newIDE/app/src/SceneEditor/index.js

```diff
@@ -674,6 +688,22 @@ export default class SceneEditor extends React.Component<Props, State> {
     });
   };
 
+  _onEditRenameObject = (objectWithContext: ?ObjectWithContext) => {
+    this.setState(
+      {
+        renamedObjectWithContext: objectWithContext,
+        selectedObjectWithContext: objectWithContext,
+      },
+      () => {
+        this.updateToolbar();
+      }
+    );
+  };
+
+  _startRenamingObject = () => {
+    this._onEditRenameObject(this.state.selectedObjectWithContext);
```

Comment:
Follow a more consistent naming for your functions. The name explains what the function does. If two functions do similar things, their names will follow the same pattern:

```suggestion
  _startRenamingSelectedObject = () => {
    this._startRenamingObject(this.state.selectedObjectWithContext);
```